eng/tools/compare-packages.ps1

```diff
@@ -0,0 +1,47 @@
+param (
+  $pathToMasterPkg,
+  $pathToCurrentPkg,
+  $dailyDevBuildNo,
+  $pathForDiffFile
+)
+
+function ExtractTGZPackages($pathToPkg){
+  $regExp = "-\d+\.\d+\.\d+(-\w*\.\d*)?(-dev-$dailyDevBuildNo)?"
+  foreach ($p in $(dir $pathToPkg -r -i *.tgz)){
+    $extractDir = (get-item $p.FullName).Directoryname + "/" + $p.BaseName
```

Comment:
I modified the script to use Join-Path and now I am creating a subfolder inside $pathtoPkg named "all-contents" where the packages are extracted and that is the subfolder I will use to diff so that it doesn't include binary ".tgz" or ".zip" files in the report